metricbeat/docs/fields.asciidoc

```diff
@@ -1586,7 +1650,7 @@ type: long
 
 
 
-*`aws.cloudwatch.namespace`*::
+*`host.cloudwatch.namespace`*::
```

Comment:
Are these changes expected? I'd say these fields should continue under `aws`.

Reply:
Not expected 😄 Thanks for catching this! This is resolved by moving the host fields into common fields.yml in libbeat.